int32 brightness
int32 contrast
int32 saturation
int32 gamma
int32 gain
int32 white_bal_temp
int32 sharpness
int32 exposure
bool auto_white_bal
bool auto_exposure
int32 resolution_width
int32 resolution_height
bool color_space
int32 power_line_frequency
bool vertical_flip
bool filter_disparity
bool filter_spots
int32 fov_start 
int32 fov_end
bool projection
int32 computation
int32 camera_height
bool fd
uint32 flags